package.json

```diff
@@ -78,7 +79,7 @@
     "build:theme-vars": "yarn run ts-node build/build_theme_vars.ts",
     "lint": "PROD=true yarn run eslint --ext js,ts,tsx --max-warnings=0 .",
     "lint:fix": "yarn run lint --fix",
-    "postinstall": "yarn run fix-canvas-deps",
+    "postinstall": "yarn run fix-canvas-deps && yarn run compile:node-fetch",
```

Comment:
Both of these introduce level of complexity that makes me question why they are **really** needed.